common/src/graph.rs

```diff
@@ -52,41 +53,52 @@ impl<N, C> Graph<N, C> {
     /// correct vertex winding.
     pub fn nearby_cubes(
         &self,
-        start: NodeId,
+        start: Position,
         distance: u32,
```

Comment:
This (and the corresponding config value in the client) should be changed to a `f32`, since it's no longer discrete.